fehm_toolkit/preprocessors/rock_properties.py

```diff
@@ -108,9 +98,9 @@ def _validate_config_all_zones_covered(config: RockPropertiesConfig, zones: tupl
     if mismatched_assignment_zones:
         raise ValueError(f'Zones in zone_assignment_order do not match those in config {mismatched_assignment_zones}')
 
-    mismatched_zones = config_zones ^ {zone.number for zone in zones}  # symmetric difference
+    mismatched_zones = config_zones - {zone.number for zone in zones}
```

Comment:
This is a functional change, I'm now allowing the case where the zone file has more zones that what we configure - this is a standard usecase in places where we append extra zones for configuration in FEHM.